cmake/Dependencies.cmake

```
@@ -93,6 +93,16 @@ if(USE_CUDA)
   endif()
 endif()
 
+# ---[ XPU
+if(USE_XPU)
+  include(${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake)
+  if(NOT PYTORCH_FOUND_XPU)
+    message(WARNING "Not compiling with XPU. Could NOT find SYCL."
+    "Suppress this warning with -DUSE_XPU=OFF.")
+    caffe2_update_option(USE_XPU OFF)
+  endif()
```

This would be printed by default every time one invokes build on Linux right now, which feels wrong, as at the moment this code is not at all usable. Please remove the warning until this is functionally complete
```suggestion
  include(${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake)
  if(NOT PYTORCH_FOUND_XPU)
    # message(WARNING "Not compiling with XPU. Could NOT find SYCL."
    # "Suppress this warning with -DUSE_XPU=OFF.")
    caffe2_update_option(USE_XPU OFF)
  endif()
```

---

ok